src/Plugins/Reminder.php

```diff
@@ -10,28 +10,28 @@ class Reminder extends NuntiusPluginAbstract {
   protected $category = 'Reminders';
 
   public $formats = [
-    '/remind me next time I logging in to (.*)/' => [
+    '/remind me next time I log into (.*)/' => [
```

Comment:
Shouldn't this be 
`remind me next time I logging in to ` because some will wrote `remind me next time I logging in to fix the light bulb`?